### DISCUSSION TITLE
FHIR Server Models

### DISCUSSION THREAD

**Discussion by masterbater** [prisma/prisma#16656] [Q&A]
Is there someone here who made their own implementation of FHIR Server specification using Prisma?

**Comment by nurul3101** [comment]
Hey @masterbater 👋 

Can you please elaborate on what you mean by implementing FHIR Server specification?
Are you referring to this tool https://www.ibm.com/products/fhir-server by IBM?

**Reply by vladi-stevanovic** [reply]
👋 Hi @masterbater, did you perhaps have a chance to check Nurul's questions? Let us know if you still need help here! 

Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!
